manifests/i/InfiniteInstant/KoduGameLab/1.6.12.0/InfiniteInstant.KoduGameLab.locale.en-US.yaml

```
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: InfiniteInstant.KoduGameLab
+PackageVersion: 1.6.12.0
+PackageLocale: en-US
+Publisher: InfiniteInstant
+PublisherUrl: http://www.infiniteinstant.com/
+# PublisherSupportUrl: 
+PrivacyUrl: https://www.kodugamelab.com/privacy/
+# Author: 
+PackageName: Kodu
+# PackageUrl: 
+License: MIT License
+LicenseUrl: https://github.com/scoy/KoduGameLab/blob/main/LICENSE.TXT
```

```suggestion
License: Proprietary
LicenseUrl: https://www.kodugamelab.com/EULA
```
License for the code is not the same as the package License